compiler/src/dmd/cond.d

```diff
@@ -659,9 +659,9 @@ extern (C++) final class VersionCondition : DVCondition
             case "AVR":
             case "BigEndian":
             case "BSD":
-            case "CppRuntime_Clang":
+            case "CppRuntime_libcpp":
             case "CppRuntime_DigitalMars":
-            case "CppRuntime_Gcc":
+            case "CppRuntime_libstdcpp":
```

Comment:
These names are equally, if not more confusing as the original (ie: libcpp is the GCC C PreProcessing library).

libstdc++v3 refers to itself as (lib)stdcxx in code.

Needs a bit more thought.

Reply:
https://github.com/dlang/dmd/pull/16613

Reply:
thanks Ian